emulator/src/core/emulator.rs

```diff
@@ -391,6 +497,77 @@ impl Emulator {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::core::ram::*;
+    use crate::kreator::assembler::Assembler;
+    use std::{
+        fs::*,
+        io::{self, Read},
+    };
+
+    fn load_asm_file(emulator: &mut Emulator, path: &str) -> io::Result<()> {
+        let mut file = File::open(path)?;
+        let mut buf = String::new();
+        file.read_to_string(&mut buf)?;
+        let asmblr = Assembler::new(&buf);
+        let mc = asmblr.assemble().expect("Fuck");
```

Comment:
Are the following Strings still intended to contain "Fuck"? It kind of looks like this was an oversight, although I don't mind the profanity if that is not the case.